activeLearner.py

```diff
@@ -10,7 +10,7 @@
 from Agent import ParameterUpdateAgent
 from replay_buffer import ParameterUpdateReplayMemory
 import pandas as pd
-
+from oracle import *
```

Comment:
It's a better practice to import only what is needed instead of all with *. Do you really need all? Please change otherwise.